cmd/connector/main.go

```diff
@@ -16,14 +16,16 @@ package main
 
 import (
 	sf "github.com/conduitio-labs/conduit-connector-salesforce"
+	sfDesinationPubsub "github.com/conduitio-labs/conduit-connector-salesforce/destination"
```

Comment:
you don't need to override the imports.
```suggestion
	"github.com/conduitio-labs/conduit-connector-salesforce/destination"
```